chidioguejiofor/volunteersng-api
# Runo: Sets up CircleCI

#### What does this PR achieve?
- Sets up continuous integration with circleci
- Sets up database image in circleci config
- Installs cross-env to account for cross-plaform compatability
- ensures all builds passes

#### Manual Testing Steps
- fetch this branch 
- run `git checkout ch-runo-circleci_config`
- Run `npm run test` to run tests locally 
- Run `git push`
- check [jobs in circleci] (https://circleci.com/gh/chidioguejiofor/) to see build results

#### Relevant Trello Tickets
- [Set Circle Ci and coveralls, code-climate](https://trello.com/c/Aea2u7l7/9-set-circle-ci-and-coveralls-code-climate)
